src/constants/indicators.js

```diff
@@ -179,59 +179,89 @@ export default {
     contextual: false,
     source: 'Education attainment - Completed Year 12, % of population',
   },
+  educationalAttainmentCertificate: {
+    name: 'Educational attainment - Certificate',
+    shortDescription: 'TODO',
+    longDescription: 'TODO',
+    dataType: DATA_TYPES.NUMBER,
+    aggregationMethod: AGGREGATION_METHODS.MEDIAN,
+    format: '0[.]0%',
+    contextual: false,
+    source: 'Education attainment - Completed Certificate III or IV, % of population',
+  },
+  educationalAttainmentTertiary: {
+    name: 'Educational attainment - Tertiary',
```

Comment:
Little 't' for Tertiary

Reply:
And maybe-but-probably-not worth adding to the readme "All text should be sentence case"

Reply:
Or just link to the content guide ?

Reply:
There's a content guide?